src/main/ui/src/components/Header/Button.css

```diff
@@ -0,0 +1,12 @@
+.btn-primary {
```

Comment:
Let's remove this and use the bootstrap styling that will match emily's buttons.
<img width="636" alt="Screen Shot 2022-09-02 at 11 31 23 AM" src="https://user-images.githubusercontent.com/7377390/188198723-46f2b7f1-d6e9-4c47-a1e3-d3c63a5e0815.png">

Also it doesn't line up. Maybe that will be fixed when using the default bootstrap?